mmtrack/models/aggregators/selsa_aggregator.py

```diff
@@ -48,18 +48,20 @@ def forward(self, x, ref_x):
             Tensor: The aggregated features of key frame proposals with shape
             [N, C].
         """
-        roi_n = x.shape[0]
-        ref_roi_n = ref_x.shape[0]
+        roi_n, C = x.shape
+        ref_roi_n, _ = ref_x.shape
+        num_c_per_att_block = C // self.num_attention_blocks
 
         x_embed = self.fc_embed(x)
         # [num_attention_blocks, roi_n, C / num_attention_blocks]
         x_embed = x_embed.view(roi_n, self.num_attention_blocks,
-                               -1).permute(1, 0, 2)
+                               num_C_per_attention_block).permute(1, 0, 2)
```

Comment:
All `num_C_per_attention_block` should be renamed to `num_c_per_att_block`
By the way, it is better to git commit in your local machine firstly. Then push it to the PR. The reason is that using `commit suggestion` may cause lint error.

Reply:
Thanks for the dedication and explanation.